docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -15,7 +15,102 @@ Lambda expressions are written with ``->``::
     x -> CAST(x AS JSON)
     x -> x + TRY(1 / 0)
 
+Examples
```

Comment:
In the above we should add some info on what functions support Lambda expressions so that in the examples we can see that it works as argument in the transform function ... otherwise there is a gap between the docs here and the examples

Reply:
And maybe we link to the transform function here or in the examples.

Reply:
I've added a direct link to the higher order function used in each of the examples.